tfx/orchestration/kubeflow/proto/kubeflow.proto

```diff
@@ -15,6 +15,8 @@ syntax = "proto3";
 
```

Comment:
Can you rename this "kfp_component_spec.proto and also put this in `tfx/extensions/experimental/kfp_compatibility/proto`?